gradle.properties

```diff
@@ -14,7 +14,7 @@ projectUrl=https://micronaut.io
 githubSlug=micronaut-projects/micronaut-security
 developers=James Kleeh,Sergio del Amo
 
-githubCoreBranch=3.6.x
+githubCoreBranch=3.5.x
```

Comment:
this is a bug fix. lets keep 3.5.x

Reply:
I set it __*Back*__ to 3.5.x?

Reply:
never mind. I am not thinking straight.I changed the PR base to 3.6.x